mlflow/tracing/processor/inference_table.py

```diff
@@ -83,7 +83,7 @@ def on_start(self, span: OTelSpan, parent_context: Optional[Context] = None):
                 timestamp_ms=span.start_time // 1_000_000,  # nanosecond to millisecond
                 execution_time_ms=None,
                 status=TraceStatus.IN_PROGRESS,
-                request_metadata={},
+                request_metadata={TRACE_SCHEMA_VERSION_KEY: str(TRACE_SCHEMA_VERSION)},
```

Comment:
do we have an example inference table with the trace schema in request_metadata? let's test that to be safe (though it should work)